docs/controllers/services/examples/http-nginx-with-healthcheck-numeric-values.yml

```diff
@@ -11,15 +11,16 @@ metadata:
 spec:
   type: LoadBalancer
   selector:
-    app: nginx-example
+    matchLabels:
+      app: nginx-example
   ports:
     - name: http
       protocol: TCP
       port: 80
       targetPort: 80
 
 ---
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
```

Comment:
I think it's an odd artefact of Github to display an update to the API version line here: the [current master does not seem to contain any further references to `extensions/v1beta1` other than for previously released manifests](https://github.com/digitalocean/digitalocean-cloud-controller-manager/search?p=1&q=extensions%2Fv1beta1&unscoped_q=extensions%2Fv1beta1) (i.e., non-examples) which I believe are best left untouched.